packages/manager/cypress/e2e/core/account/account-linode-managed.spec.ts

```diff
@@ -0,0 +1,181 @@
+} from 'support/constants/account';
+import { mockGetProfile } from 'support/intercepts/profile';
+import { ui } from 'support/ui';
+import {
+  visitUrlWithManagedDisabled,
+  visitUrlWithManagedEnabled,
+} from 'support/api/managed';
+
+describe('Account Linode Managed', () => {
+  /*
+   * - Confirms that a user can add linode managed from the Account Settings page.
+   * - Confirms that user is told about the Managed price.
+   * - Confirms that Cloud Manager displays the Managed state.
+   */
+  it.only('users can enable Linode Managed', () => {
+    const mockAccount = accountFactory.build();
+    const mockProfile = profileFactory.build({
+      username: 'mock-user',
+      restricted: false,
+    });
+    const mockLinodes = new Array(4).fill(null).map(
+      (item: null, index: number): Linode => {
+        return linodeFactory.build({
+          label: `Linode ${index}`,
+          region: chooseRegion().id,
+        });
+      }
+    );
+
+    mockGetLinodes(mockLinodes).as('getLinodes');
+    mockGetAccount(mockAccount).as('getAccount');
+    mockGetProfile(mockProfile).as('getProfile');
+    mockEnableLinodeManaged().as('enableLinodeManaged');
+
+    // Navigate to Account Settings page, click "Add Linode Managed" button.
+    visitUrlWithManagedDisabled('/account/settings');
+    cy.wait(['@getAccount', '@getProfile', '@getLinodes']);
+
+    ui.button
+      .findByTitle('Add Linode Managed')
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+
+    ui.dialog
+      .findByTitle('Just to confirm...')
+      .should('be.visible')
+      .within(() => {
+        cy.get('h6')
+          .invoke('text')
+          .then((text) => {
+            console.log(`h6 text: ${text.trim()}`);
+            expect(text.trim()).to.equal(
+              linodeEnabledMessageText(mockLinodes.length)
+            );
+          });
+
+        // Confirm that submit button is enabled.
+        ui.button
+          .findByTitle('Add Linode Managed')
+          .should('be.visible')
+          .should('be.enabled')
+          .click();
+      });
+
+    cy.wait('@enableLinodeManaged');
+
+    // Confirm that Cloud Manager displays a notice about Linod managed is enabled.
+    cy.findByText(linodeManagedStateMessageText, { exact: false }).should(
+      'be.visible'
+    );
+  });
+
+  /*
+   * - Confirms Cloud Manager behavior when a restricted user attempts to enable Linode Managed.
+   * - Confirms that API error response message is displayed in confirmation dialog.
+   */
+  it('restricted users cannot cancel account', () => {
+    const mockAccount = accountFactory.build();
+    const mockProfile = profileFactory.build({
+      username: 'mock-restricted-user',
+      restricted: true,
+    });
+    const errorMessage = 'Unauthorized';
+
+    mockGetAccount(mockAccount).as('getAccount');
+    mockGetProfile(mockProfile).as('getProfile');
+    mockEnableLinodeManagedError(errorMessage, 403).as('enableLinodeManaged');
```

Comment:
```suggestion
    mockGetLinodes([]);
    mockGetAccount(mockAccount).as('getAccount');
    mockGetProfile(mockProfile).as('getProfile');
    mockEnableLinodeManagedError(errorMessage, 403).as('enableLinodeManaged');
```
Let's mock the Linode GET request to correspond with what a real request would look like for restricted users